webserver/webserver/src/main/java/io/helidon/webserver/WebTracingConfig.java

```diff
@@ -0,0 +1,307 @@
+/*
+ * Copyright (c) 2019 Oracle and/or its affiliates. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.helidon.webserver;
+
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+import java.util.stream.Collectors;
+
+import io.helidon.common.CollectionsHelper;
+import io.helidon.common.context.Context;
+import io.helidon.common.context.Contexts;
+import io.helidon.common.http.Http;
+import io.helidon.config.Config;
+import io.helidon.tracing.config.SpanTracingConfig;
+import io.helidon.tracing.config.TracingConfig;
+import io.helidon.tracing.config.TracingConfigUtil;
+
+import io.opentracing.Span;
+import io.opentracing.SpanContext;
+import io.opentracing.Tracer;
+import io.opentracing.propagation.Format;
+import io.opentracing.propagation.TextMapExtractAdapter;
+import io.opentracing.tag.Tags;
+import io.opentracing.util.GlobalTracer;
+
+/**
+ * Tracing configuration for webserver.
+ * Tracing configuration has two components - an overall (application wide) {@link io.helidon.tracing.config.TracingConfig}
+ *  and a path specific {@link PathTracingConfig}.
+ */
+public abstract class WebTracingConfig {
+    /**
+     * Environment tracing configuration.
+     *
```

Comment:
Maybe just delete this.

Reply:
The method is needed, some documentation should be present. Reworded and added details